### ISSUE TITLE
!anything doesn't work in included files

### ISSUE THREAD

**Issue by eht16** [taverntesting/tavern#192]
When I try to use `!anything` in a file included in a test, I get the following error:
```
    - Value mismatch in body: Type of returned data was different than expected (expected["roles"] = '<Tavern YAML sentinel for anything>', actual["roles"] = '['role-foo-bar']')
```
However, if I use the same test stage directly without imports, the `!anything` works as expected.

Full example (server is based on the examples and works the same way):
https://gist.github.com/eht16/297ca039aa69cfc7bcb4ddb8732b3f4f

```
platform linux -- Python 3.7.0, pytest-3.8.1, py-1.6.0, pluggy-0.7.1 -- /home/enrico/tavern/.tox/py37/bin/python3.7
cachedir: .pytest_cache
rootdir: /home/enrico/tavern, inifile: tox.ini
plugins: tavern-0.18.2
```

**Comment by michaelboulton** [comment]
It seems like this is due to deepcopy creating a copy of `ANYTHING` from util/loader.py when it's in an included test when it should return the same object

**Comment by michaelboulton** [comment]
This should be fixed in 0.18.3.
